ballerina/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "file"
-version = "1.2.1"
+version = "1.2.2"
```

Comment:
Shall we change this to the next minor release?. We need to do minor release on top of update1

Reply:
If all the breaking changes are reverted, should we update to update 1 dependency?

Reply:
If it is required only we need to release it on top of update1. any this is an API addition. better we go with minor release